src/function-prewarm.ts

```diff
@@ -6,7 +6,9 @@ export interface PrewarmClientInitializer<T, O> {
   init: (key: string, props?: PrewarmProps) => O;
 }
 
+// TODO: use names that align with the AWS.SDK
 export type ClientName =
+  | "SECRETS_MANAGER"
```

Comment:
100%, do this or create issue?

Reply:
I can just do it if we think it's not worth being in its own PR? My only thought is that it will affect #405

Reply:
going ahead with this change.

Reply:
Done.